### PR TITLE
Add drawing module

### PR DESCRIPTION
**Pull request by jazzpi**
Adds the option to do drawings with Wololobot without using Florins.
This adds the following commands:

                  Command | Description
--------------------------|---------------------------------------------------
             `!draw open` | Opens a drawing
 `!draw close `*`number`* | Closes the drawing and chooses *`number`* winners
`!draw reroll `*`winner`* | Chooses a different entrant instead of *`winner`*
                  `!play` | Enter the drawing
               `!winners` | Displays the winners of the drawing

The `drawing` module takes an optional argument `opts` with the attributes `subChances` and `normalChances`. Subscribers to the channel are entered `subChances` times into the drawing, non-subscribers are entered `normalChances` times.
`subChances` defaults to `2`, `normalChances` defaults to `1`.

### CODE REVIEW

File: src/modules/drawing.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      bot.send('New drawing opened! To enter, type !play. Subscribers have double the chances to' +
+               'win. Type !draw close <number of players> to end the drawing.')
+    })
+
+    bot.command('!draw close'
+                , { rank: 'mod' }
+                , (message, number) => {
+      open = false
+      number = parseInt(number)
+      if (number.isNaN) {
```
Comment (goto-bus-stop) [inline src/modules/drawing.js]: `isNaN(number)`

File: src/modules/drawing.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      }
+    })
+
+    bot.command('!draw open'
+                , { rank: 'mod' }
+                , (message) => {
+      entrants = []
+      open = true
+      nEntrants = 0
+      bot.send('New drawing opened! To enter, type !play. Subscribers have double the chances to' +
```
Comment (goto-bus-stop) [inline src/modules/drawing.js]: minor: it's not really always double with configurable chances. maybe `${n} times`  or so (or you can just not care)

**Comment by jazzpi** [comment]
The `number.isNaN` thing happens more often than I'd like to admit... :neutral_face:

File: src/modules/drawing.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    })
+
+    bot.command('!draw open'
+                , { rank: 'mod' }
+                , (message) => {
+      entrants = []
+      open = true
+      nEntrants = 0
+      bot.send('New drawing opened! To enter, type !play. Subscribers have ' +
+               '${opts.subChances / opts.normalChances}x higher chances to' +
```
Comment (goto-bus-stop) [inline src/modules/drawing.js]: should be a backtick-ed string (...because javascript, i guess)

**Comment by goto-bus-stop** [comment]
hehe :shipit:

**Pull request merged**